sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -19,7 +19,7 @@
 
 ### Breaking Changes
 * Azure AI Foundry Project endpoint is now required to construct the `AIProjectClient`. It has the form
-`https://<your-ai-services-account-name>.services.ai.azure.com/api/projects/<your-project-name>`. Find it in your AI Foundry Project Overview page. 
+`https://<your-ai-services-account-name>.services.ai.azure.com/api/projects/<your-project-name>`. Find it in your AI Foundry Project Overview page. The support of connection strings as the project endpoints and hub-based projects was dropped. Please create a new Azure AI Foundry resource, which uses endpoints (recommended), or pin the version of `Azure.AI.Projects` to `1.0.0-beta.8` if it is not possible.
```

Comment:
Little rephrase suggestion, could you change to notes like below?

Support for connection strings as project endpoints and hub-based projects has been discontinued. We recommend creating a new Azure AI Foundry resource utilizing endpoints. If this is not possible, please pin the version of `Azure.AI.Projects` to version `1.0.0-beta.8` or earlier.